### ISSUE TITLE
Talon Does Not Recognize Screen Reader Keyboard Shortcuts That Use Nvda Keys 

### ISSUE THREAD

**Issue by tararoys** [talonvoice/talon#536]
Talon Does Not Recognize Screen Reader Keyboard Shortcuts That Use Nvda Keys

The screen reader NVDA gives the user the option to use either the insert key as "NVDA key" or the CapsLock key as its "NVDA key"
I have only tested the insert key since I do not know how to use the capslock key in talon.

 The NVDA key is critical to most NVDA screen reader function

# To reproduce the bug use the following talon code

With NVDA active, you can use the following Talon code to test it's functionality: 

 it must be noted that you should use this only with headphones because having you screen reader read things out loud to you  Via speakers will cause  talon, your voice control software, to do arbitrary things to your computer.

The following talon code does not work To launch NVDA  screen reader settings dialogue:

```
nevada settings:
    key(insert-ctrl-v)
```

# to  create a workaround  use the following talon code

The following workaround in talon code does work  to launch NVDA screen reader setting dialogue.

```
nevada settings:
   key(insert:down ctrl-v insert:up) 
```

The following fragile work around can also be used for arbitrary  NVDA screen reader codes:

```
nevada hold:
    # Holds down the NVDA Key, which is the insert key.
    clip.set_text("insert key is down. Release with command nevada release")
    sleep(100ms)
    key(insert:down c insert:up)
    sleep(6000ms)
    key(insert:down)
```

```
nevada release:
    # Holds up the NVDA Key, which is the insert key.
    key(insert:up)
    clip.set_text("The insert key is released.")
    sleep(100ms)
    key(insert:down c insert:up)

```
 this work around is fragile because it requires the user to remember to release NVDA key after you are done giving the keyboard shortcut command.
if you forget to specifically release the NVDA key,   any further keys that you press will be arbitrary NVDA keyboard shortcuts.
in an attempt to make this less fragile,  I use NVDA ability to read items from the clipboard  with the shortcut NVDA-c to paste a documentation into the clipboard which NVDA will read allowed to indicate to that the NVDA key is being held down.
 This has its own drawbacks because this means anything on the clipboard is erased.

**Comment by lunixbochs** [comment]
You can only press modifier keys (such as ctrl, alt, etc) in the `key(a-b-c)` style. Insert is not a modifier key, so you are correct about needing to `key(insert:down a insert:up)`. This isn't a Talon bug, and I don't currently plan on changing it.
